portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -77,7 +109,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    * the Gson library dependency to pom.xml.
    * @params {List}
```

Comment:
This is not how Java doc comments are written. 
```suggestion
   * @params comments
```
We don't need a type here because Java is strongly typed.